Calendar_project/clock.js

```diff
@@ -0,0 +1,24 @@
+const clockContainer = document.querySelector(".js-clock");
+const clockTitle = clockContainer.querySelector("h1");
+
+
+function getTime() {
+    const nowTime = new Date();
+    const minutes = nowTime.getMinutes();
+    const hours = nowTime.getHours();
+    const seconds = nowTime.getSeconds();
+    clockTitle.innerText = `${hours < 10 ? `0${hours}` : hours}:${minutes < 10 ? `0${minutes}` : minutes
+        }:${seconds < 10 ? `0${seconds}` : seconds}`;
```

Comment:
템플릿 리터럴로 잘 작성해주셨네요!👍
한줄이 너무 길어진다면 아래와 같이 hours, minutes, seconds가 구분되게 개행해줘도 좋을 것 같습니다.
```suggestion
    clockTitle.innerText = `${hours < 10 ? `0${hours}` : hours
      }:${minutes < 10 ? `0${minutes}` : minutes
      }:${seconds < 10 ? `0${seconds}` : seconds}`;
```